src/vivarium_gates_nutrition_optimization/constants/data_values.py

```diff
@@ -105,7 +105,7 @@ class _HemoglobinDistributionParameters(NamedTuple):
         paths.HEMOGLOBIN_PREGNANCY_ADJUSTMENT_FACTORS_CSV, index_col=0
     ).squeeze(),
     data_keys.HEMOGLOBIN.STANDARD_DEVIATION: pd.Series(
-        np.repeat(1.032920188, 1000), [f"draw_{i}" for i in range(1000)]
+        np.repeat(1.032920188, 500), [f"draw_{i}" for i in range(500)]
```

Comment:
We'll want to use vi_globals.NUM_DRAWS instead of 500 throughout the PR.

Reply:
Updated here and in a few additional places throughout.